packages/venia-ui/lib/queries/userSubscription.graphql

```diff
@@ -0,0 +1,9 @@
+# expects bearer header to be set via context to return data
+mutation userSubscription($is_subscribed: Boolean!) {
+    updateCustomer(input: { is_subscribed: $is_subscribed }) {
+        customer {
+            id
+            is_subscribed
```

Comment:
Here you would re-use the Fragment used for the initial query. Example can be found [here](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/guestForm.gql.js) with how we colocate them. A dedicated fragments file might be overkill here, so feel free to put everything in a `communicationsPage.gql.js` file.